eas.json

```diff
@@ -23,7 +23,8 @@
       "distribution": "internal",
       "ios": {
         "buildConfiguration": "Debug",
-        "simulator": false
+        "simulator": false,
+        "resourceClass": "m1-experimental"
```

Comment:
This works just fine!

 If we wanted, I think we can drop `resourceClass` from dev, preview etc because they inherit from the `production` profile. Probably worth it to run a test without it here and see if it passes it along properly.

Reply:
Ok, I had to update the CLI, but it works to have it in production and just inherit 👍